PDF_UA/2/8.2 Logical structure/8.2.5 Additional requirements for specific structure types/8.2.5.8 Table of Contents (TOC and TOCI)/verapdf-profile-8-2-5-8-t01.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFUA_2">
+    <details creator="veraPDF Consortium" created="2023-03-13T10:58:04.137+03:00">
+        <name>ISO 14289-2:202x - 8.2 Logical structure - 8.2.5.8 Table of Contents (TOC/TOCI) - Reference in TOCI</name>
+        <description>Each TOCI in the table of contents shall identify the target of the reference using the Ref entry, 
+            either directly on the TOCI structure element itself or on a child structure element contained within</description>
+    </details>
+    <hash></hash>
+    <rules>
+        <rule object="SETOCI">
+            <id specification="ISO_14289_2" clause="8.2.5.8" testNumber="1"/>
+            <description>Each TOCI in the table of contents shall identify the target of the reference using the Ref entry,
+                either directly on the TOCI structure element itself or on a child structure element contained within</description>
+            <test>containsRef == true</test>
+            <error>
+                <message>TOCI in the TOC does not contain the Ref entry, neither directly on the TOCI structure element 
```

Comment:
TOCI in the TOC does not contain the Ref entry, neither directly on the TOCI structure element itself nor on its descendant structure elements